stock_release_channel_plan_process_end_time/tests/test_stock_release_channel_plan.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+import pytz
+
+from odoo import fields
+
+from odoo.addons.partner_tz.tools.tz_utils import tz_to_utc_naive_datetime
+from odoo.addons.stock_release_channel_plan.tests import test_stock_release_channel_plan
+from odoo.addons.stock_release_channel_process_end_time.utils import float_to_time
+
+
+class TestStockReleaseChannelPlan(
+    test_stock_release_channel_plan.TestStockReleaseChannelPlan
```

Comment:
hmm I'm not in favor of inheriting full test classes as every test you add there will run twice here.
Also overriding the method and calling super seems weird...
Can't you add a mixin and reuse it here?